### DISCUSSION TITLE
Improving contour selection for an EBSD map

### DISCUSSION THREAD

**Discussion by lindasquillaci** [mtex-toolbox/mtex#1888] [Ask Anything] [answered]
Hi!

I would like to get some help understanding how to avoid artefacts when plotting the grain boundaries of the EBSD map contained in fig6:

Fig 6
![Fig6](https://github.com/mtex-toolbox/mtex/assets/138127555/3dad69e3-1522-4145-a1d2-0a54352d9b0e)

Fig 7
![Fig7](https://github.com/mtex-toolbox/mtex/assets/138127555/67dda1cf-d728-4817-a8a9-b354420e9cb6)

I am working with titanium alloys that present very fine alpha grains (martenisite) and when reconstructing the parent beta phase (fig 6) I get a false contour for the grain boundaries. Can someone help me see why this happens? Is that to do with non indexed pixels? How do I tell Mtex to plot the outside contours of fig 7?

Thank you in advance for any input!
Regards,
Linda

**Comment by kilir** [comment, marked as answer]
Hi,
try `calcGrains` with the option `boundary','tight'` eventually with a numeric value.
Cheers,
Rüdiger
